dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -155,3 +157,8 @@
       });
     }
   });
+
+  $(document).ready(function() {
```

Comment:
I'm not sure if this is the preferred way to put React content into a non-React page - ideally we'd eventually pull all account-related stuff into React, but that seems like a much larger task